appveyor.yml

```diff
@@ -140,11 +145,6 @@
 
                 cd build
 
-                # Make sure we have Python27-64 before any other version; Python path per https://www.appveyor.com/docs/installed-software#python
```

Comment:
Yep, looks like both Python and the C++ builds need this in their 64-bit config. Spinning a new version